notebooks/en/semantic_reranking_elasticsearch.ipynb

```diff
@@ -0,0 +1,609 @@
+      "2024-08-13 17:04:18,789 INFO : Creating model with id 'cross-encoder__ms-marco-minilm-l-6-v2'\n",
+      "2024-08-13 17:04:21,123 INFO : Uploading model definition\n",
+      "100% 87/87 [00:55<00:00,  1.57 parts/s]\n",
+      "2024-08-13 17:05:16,416 INFO : Uploading model vocabulary\n",
+      "2024-08-13 17:05:16,987 INFO : Starting model deployment\n",
+      "2024-08-13 17:05:18,238 INFO : Model successfully imported with id 'cross-encoder__ms-marco-minilm-l-6-v2'\n"
+     ]
+    }
+   ],
+   "source": [
+    "!eland_import_hub_model \\\n",
+    "  --cloud-id $ELASTIC_CLOUD_ID \\\n",
+    "  --es-api-key $ELASTIC_API_KEY \\\n",
+    "  --hub-model-id cross-encoder/ms-marco-MiniLM-L-6-v2 \\\n",
+    "  --task-type text_similarity \\\n",
+    "  --clear-previous \\\n",
+    "  --start"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "DRIABkGAgV_Q"
+   },
+   "source": [
+    "## Create inference endpoint\n",
+    "\n",
+    "Next we'll create an inference endpoint for the `rerank` task to deploy and manage our model and, if necessary, spin up the necessary ML resources behind the scenes."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "DiKsd3YygV_Q",
+    "outputId": "c3c46c6b-b502-4167-c98c-d2e2e0a4613c"
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "ObjectApiResponse({'endpoints': [{'model_id': 'my-msmarco-minilm-model', 'inference_id': 'my-msmarco-minilm-model', 'task_type': 'rerank', 'service': 'elasticsearch', 'service_settings': {'num_allocations': 1, 'num_threads': 1, 'model_id': 'cross-encoder__ms-marco-minilm-l-6-v2'}, 'task_settings': {'return_documents': True}}]})"
+      ]
+     },
+     "execution_count": 20,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "client.inference.put(\n",
+    "    task_type=\"rerank\",\n",
+    "    inference_id=\"my-msmarco-minilm-model\",\n",
+    "    inference_config={\n",
+    "        \"service\": \"elasticsearch\",\n",
+    "        \"service_settings\": {\n",
+    "            \"model_id\": \"cross-encoder__ms-marco-minilm-l-6-v2\",\n",
+    "            \"num_allocations\": 1,\n",
+    "            \"num_threads\": 1,\n",
+    "        },\n",
+    "    },\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Run the following command to confirm your inference endpoint is deployed."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "client.inference.get()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "⚠️ When you deploy your model, you might need to sync your ML saved objects in the Kibana (or Serverless) UI.\n",
```

Comment:
ℹ️ This is the main potential stumbling block in this notebook. Please let me know if having trouble in UI getting this sorted out, or if could make the steps for fixing it clearer.